htsget.md

```diff
@@ -41,9 +41,13 @@ JSON responses SHOULD include a `Content-Type` header describing the htsget prot
 
     Content-Type: application/vnd.ga4gh.htsget.v0.2rc+json; charset=utf-8
 
-## Authentication
+## Security
 
-Requests to the retrieval API endpoint may be authenticated by means of an OAuth2 bearer token included in the request headers, as detailed in [RFC 6750]. Briefly, the client supplies the header `Authorization: Bearer xxxx` with each HTTPS request, where `xxxx` is a private token. The mechanisms by which clients originally obtain their authentication tokens, and by which servers verify them, are currently beyond the scope of this specification. Servers may honor non-authenticated requests at their discretion.
+This data retrieval API enables the retrieval of highly sensitive genomic data by means of a client/server model.  Effective security measures are essential to protect the integrity and confidentiality of these data.  
+
+All communication channels used to transmit sensitive information, including security-critical information and genomic data, MUST BE protected using Transport Level Security (TLS) version 1.2 or later, as specified in [RFC 5246]. 
+
+A precondition of this specification is that the identity of all end users and client software have been authenticated and access authorized in accordance with the policies enforced by the applicable data holder (including open-access policies).  GA4GH recommends the use of OAuth 2.0 ([RFC 6749]) and its [OpenID Connect] profile for these purposes.  Details regarding specific implementations of these standards are beyond the scope of this specification.  
```

Comment:
Reads like open access should be authenticated and authorized. Is this the intention? Why not make the requrement for authentication and authorization optional for open access data?

Reply:
I'd suggest to clarify and change "the use of OAuth 2.0 ([RFC 6749]) and its [OpenID Connect] profile for these purposes." to "the use of OAuth 2.0 ([RFC 6749]) for authorization and OpenID Connect for authentication purposes."

Reply:
Re open access, "...in accordance with policies enforced by the applicable data holder (including open-access policies)" should make it clear that if a data holder's policy says that anyone may access the data and that neither authentication nor authorization is required, then neither is needed.  No change needed.

Reply:
Re OpenID Connect comment, OpenID Connect is in fact a profile of OAuth 2.0 and is directly dependent upon RFC 6749 (OAuth 2.0 framework).  Correct as written.  No change is needed.

Reply:
Although it can be argued that the current language regarding open-access policies is sufficient, it can and should be clarified so that one does not need to analyze what the parenthetical "including open-access policies" does or does not constrain. Suggestion

> A precondition of this specification is that the identity of all end users and client software have been authenticated and access authorized ***if*** required by policies of the applicable data holder.

Reply:
I agree with Mike's suggestion. It is a little less cryptic than the text in parenthesis.